Marlin/language_de.h

```diff
@@ -261,11 +270,14 @@
   #define MSG_UBL_INVALIDATE_ALL            _UxGT("Alles annullieren")
   #define MSG_UBL_INVALIDATE_CLOSEST        _UxGT("Nächstlieg. ann.")
   #define MSG_UBL_FINE_TUNE_ALL             _UxGT("Feineinstellung Alle")
-  #define MSG_UBL_FINE_TUNE_CLOSEST         _UxGT("Feineinst- Nächstl.")
+  #define MSG_UBL_FINE_TUNE_CLOSEST         _UxGT("Feineinst. Nächstl.")
   #define MSG_UBL_STORAGE_MESH_MENU         _UxGT("Netz Speicherplatz")
   #define MSG_UBL_STORAGE_SLOT              _UxGT("Memory Slot")
   #define MSG_UBL_LOAD_MESH                 _UxGT("Bett Netz laden")
   #define MSG_UBL_SAVE_MESH                 _UxGT("Bett Netz speichern")
+  #define MSG_UBL_SAVE_ERROR                _UxGT("save_UBL_active() Fehler")
+  #define MSG_UBL_RESTORE_ERROR             _UxGT("restore_UBL_active() Fehler")
```

Comment:
I expected this as a placeholder already, but, like mentioned, this whole UBL stuff is out of my reach :)